src/anupSJun20/CharacterMethods.java

```diff
@@ -0,0 +1,39 @@
+package anupSJun20;
+/*1) Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+Input: 1rRpd3F9#K(E
+Output : 
+Total digit: 3
+Total Letters: 7
+Total Uppercase: 4
+Total Lowercase: 3
+Total Special char: 2*/
+
+public class CharacterMethods {
+	static void findCharCountInString(String str) {
+		int digitCount = 0;
+		int uppercase = 0;
+		int lowercase = 0;
+		int specialChar = 0;
+		for(int index = 0;index<str.length();index++) {
+			char ch = str.charAt(index);
+			if (Character.isDigit(ch)==true)
+				digitCount++;
+			else if(Character.isLetter(ch) == true) {
+			if(Character.isUpperCase(ch) == true)
```

Comment:
First thing why are you  here writing == true, because if a condition will be true then only it will go in the respective block. 
Second thing else if(Character.isLetter(ch) == true) this if block is not needed. 
Because it will already going to cover under uppercase and lowercase.